tests/roles/dataplane_adoption/tasks/octavia_certs.yaml

```diff
@@ -0,0 +1,83 @@
+- name: Migrate old Octavia certificates to new dual CA structure
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    CONTROLLER1_SSH="{{ controller1_ssh }}"
+    CONTROLLER1_SCP="{{ controller1_ssh  | regex_replace('^ssh', 'scp')}}"
+    SERVER_CA_PASSPHRASE=$($CONTROLLER1_SSH grep ^ca_private_key_passphrase /var/lib/config-data/puppet-generated/octavia/etc/octavia/octavia.conf)
+    export SERVER_CA_PASSPHRASE=$(echo "${SERVER_CA_PASSPHRASE}"  | cut -d '=' -f 2 | xargs)
+    export CLIENT_PASSPHRASE="ThisIsOnlyAppliedTemporarily"
+    CERT_SUBJECT="/C=US/ST=Denial/L=Springfield/O=Dis/CN=www.example.com"
+    CERT_MIGRATE_PATH="$HOME/octavia_cert_migration"
+
+    mkdir -p ${CERT_MIGRATE_PATH}
+    cd ${CERT_MIGRATE_PATH}
+    # Set up the server CA
+    mkdir -p server_ca
+    cd server_ca
+    mkdir -p certs crl newcerts private csr
+    chmod 700 private
+    ${CONTROLLER1_SCP}:/var/lib/config-data/puppet-generated/octavia/etc/octavia/certs/private/cakey.pem private/server_ca.key.pem
+    chmod 400 private/server_ca.key.pem
+    ${CONTROLLER1_SCP}:/tmp/octavia-ssl/client-.pem certs/old_client_cert.pem
```

Comment:
Does `/tmp/octavia-ssl/` still exist after controller reboot?

Reply:
I tried it out. At least in the standlone deployment the path and the files in it still exist after reboot